primitives/chainblocks/src/lib.rs

```diff
@@ -100,3 +102,48 @@ pub fn init() {
 
 	// cblog!("Chainblocks initialized!");
 }
+
+pub fn http_json_post(url: &str, body: &[u8]) -> Result<Vec<u8>, &'static str> {
+	log::debug!("sp_chainblocks http_request called...");
+
+	let request =
+		offchain::http_request_start("POST", url, &[]).map_err(|_| "Failed to start request")?;
+
+	offchain::http_request_add_header(request, "Content-Type", "application/json")
+		.map_err(|_| "Failed to add header")?;
+
+	offchain::http_request_write_body(request, body, None).map_err(|_| "Failed to write body")?;
+
+	// send off the request
+	offchain::http_request_write_body(request, &[], None).unwrap();
+
+	let results = offchain::http_response_wait(&[request], None);
```

Comment:
Giovanni, since "there are uncertainty in terms of how long the request will take, and the result may not be the same all the time. This causes problem for the network reaching consensus." (https://substrate.recipes/off-chain-workers/http-json.html)

Would it be wise if the deadline parameter was not `None` ?

Reply:
Note to Self: Find out how HTTP requests can cause problem for the network reaching consensus

Reply:
They run in different threads/contexts